### ISSUE TITLE
Browsing genres

### ISSUE THREAD

**Issue by praise-you** [peguerosdc/subplayer#37]
Hi Carlos,
Subplayer is a beautiful client for Navidrome. It's slick!
You have already implemented Artist, Album, Favorites and Playlist but Genres are missing. I know people browse their music in very different ways and Genres can be overlooked, but Genres are a great way to quickly find the music that appeal to one's current mood.
Thanks for your consideration,

**Comment by peguerosdc** [comment]
Hi!

Thanks for trying Subplayer! Agree, people browse their music in different ways and they all are important :) I didn't implement it in the beginning because I don't use it, but will do in the next months when I have more time.

**Comment by peguerosdc** [comment]
@praise-you sorry for the delay! I have added a view to browse the music library by genre (it is in the Sidebar and I also added a Setting to customise which browsing options are displayed).

While testing with my instance of Navidrome, I realised that I am unable to retrieve the songs of the genre `"<Empty>"` (which I assume is what Navidrome returns when that tag is not set) as the server returns an empty response. Just for you to consider if you find something similar :)

Oh, and the implementation only works with Subsonic API greater than 1.13.0 as the `songCount` field is required in the `getGenres` response. Not using that parameter (i.e. not knowing in advance how many songs to request) would require to request songs until finding a response that looks like a "last page", which I don't have plans to implement.
